kafka-client/src/main/java/dev/responsive/kafka/internal/db/rs3/RS3TableFactory.java

```diff
@@ -54,6 +49,35 @@ public RemoteKVTable<WalEntry> kvTable(
     );
   }
 
+  public RemoteWindowTable<WalEntry> windowTable(
+      final String name,
+      final ResponsiveConfig config,
+      final ResponsiveMetrics responsiveMetrics,
+      final ResponsiveMetrics.MetricScopeBuilder scopeBuilder
+  ) {
+    final var storeId = lookupStoreId(config, name);
+    final var pssPartitioner = new PssDirectPartitioner();
+    final var rs3Client = connector.connect();
+    return new RS3WindowTable(
+        name,
+        storeId,
+        rs3Client,
+        pssPartitioner,
+        responsiveMetrics,
+        scopeBuilder
+    );
+  }
+
+  private UUID lookupStoreId(ResponsiveConfig config, String name) {
+    Map<String, String> storeIdMapping = config.getMap(
```

Comment:
very minor nit: use `final` everywhere possible (we actually have a checkstyle rule for this in kafka streams, might be useful to add one here too 🤷‍♀️ )

Reply:
ditto (and especially) for the method parameters

Reply:
Yeah, old habits die hard. I'll try to fix the ones I spot.